liquidation_inspector.py

```diff
@@ -0,0 +1,105 @@
+# Internal imports
+from mev_inspect import utils
+from mev_inspect.config import load_config
+from mev_inspect.schemas.blocks import NestedTrace, TraceType
+from mev_inspect.schemas.classified_traces import Classification, ClassifiedTrace
+from mev_inspect.schemas.strategy import StrategyType, Strategy, Liquidation
+from mev_inspect.classifier_specs import CLASSIFIER_SPECS
+from mev_inspect.trace_classifier import TraceClassifier
+from mev_inspect import block
+
+# External Libraries
+import json
+import pandas as pd
+from typing import Optional
+from web3 import Web3
+from typing import List, Optional
+
+# TODO: adjust profit to new model
+#        unit test
+#        coinbase check / collateral source
+
+liquidations = []
```

Comment:
any non-constant values should be inside a function

global variables in Python are difficult to reason about - it's like sneaky passing a variable into all functions since anyone can access and modify it. Makes it difficult to know what does and doesn't depend on it, and how the values changed

They also make it difficult to scale code - running two processes / instances of this code will create two separate global variables while the code itself might still be assuming it's one shared one.